tests/testthat/base-comparison.r

```diff
@@ -0,0 +1,10 @@
+context("Assertion messages on base operators")
```

Comment:
This won't get run because the test filename doesn't start with `test`.

Generally, please match the test file name to the R file name: http://style.tidyverse.org/tests.html#organisation

Reply:
Yep sorry, my mistake